src/Microsoft.ML.PipelineInference/AutoInference.cs

```diff
@@ -524,6 +431,18 @@ public void AddEvaluated(PipelinePattern pipeline)
                     d += 1e-3;
                 _sortedSampledElements.Add(d, pipeline);
                 _history.Add(pipeline);
+
+                using (var ch = _host.Start("Suggested Pipeline"))
+                {
+                    ch.Info($"PipelineSweeper Pipeline Id : {pipeline.UniqueId}");
```

Comment:
Can we also print an increasing pipeline number? This lets the user see the progress of the sweeping.